image_generation/latent_consistency_model/OVLatentConsistencyModelPipeline.py

```diff
@@ -0,0 +1,241 @@
+from typing import Union, Optional, Any, List, Dict
+from transformers import CLIPTokenizer, CLIPImageProcessor
+from diffusers.pipelines.stable_diffusion.safety_checker import (
+    StableDiffusionSafetyChecker,
+)
+from diffusers.pipelines.stable_diffusion import StableDiffusionPipelineOutput
+from diffusers.image_processor import VaeImageProcessor
+from diffusers import DiffusionPipeline
```

Comment:
I think the purpose of Python examples in this repository is to demonstrate how to work with the models without extra dependencies in almost native Python+openvino environment. Here we are importing some stuff with quite heavy dependencies like `torch`. It should be only applicable for code that does export of the models, but not the inference code.

Reply:
Noted, thank you! I'm reviewing the unnecessary dependencies and will take these out in the next commit.

Reply:
@slyalin, I had two follow-up questions please: 

1. I removed the CLIPTokenizer and Image Processor dependencies. The other dependencies are being used in the inference pipeline. Would it make sense to export the VaeImageProcessor model offline, since I notice this is an API for input prep?

3. I see the tokenizer export is in the llm benchmarking part of the repo - is it useful to replace LCM's original tokenizer with the OVtokenizer, or is this not the right time? Thank you!
```
#Define LCM scheduler, tokenizer
#TBD - access pre-saved scheduler and tokenizer
pipe = DiffusionPipeline.from_pretrained("SimianLuo/LCM_Dreamshaper_v7")
scheduler = pipe.scheduler
tokenizer = pipe.tokenizer
```